js/src/personal/riyuan/components/project/Project.tsx

```diff
@@ -1,12 +1,9 @@
-import { Text, Title, Container } from '@mantine/core'
+import { Container, Title, Text } from '@mantine/core'
 import styles from '@/personal/riyuan/Riyuan.module.css'
 
-export function Project() {
+export function RiyuanProjectsPage() {
```

Comment:
File name should match component name.
We may want to standardize on RiyuanProjects.page.tsx though, to denote top level react components.

Reply:
done